FNF_MissionTemplate.VR/server/init/fn_serverSafety.sqf

```diff
@@ -22,16 +22,24 @@ call phx_fnc_handleSafetyVics; //Make vehicles invincible until safety ends
 }] call CBA_fnc_waitUntilAndExecute;
 
 [{f_var_mission_timer < 0}, {
-  missionNamespace setVariable ["phx_safetyEnabled",false,true];
-  missionNamespace setVariable ["phx_safetyEndTime", round CBA_missionTime, true];
-  ["SafeStartMissionStarting",["Mission starting now!"]] remoteExec ["bis_fnc_showNotification",0,false];
-  ["off"] call acex_fortify_fnc_handleChatCommand;
-
-  [] call phx_fnc_webhook_roundStart;
-
-  {
-    if !(getMarkerColor _x isEqualTo "") then {
```

Comment:
Correct me if I'm wrong, but this check appeared useless so I removed it.

Reply:
the same check is done in fn_server_setupGame.sqf and fn_hideMarkers.sqf so beleive it is likely legacy code from when markers were being messed with before, @Mjolnir64 might know what it was for

Reply:
Yeah I saw that. But searching the rest of the code, I didn't see any cases where the safe start markers get a "" color set. Hence my assumption these checks are dead code.

Reply:
> Markers that have been deleted or don't exist return a color of "", so it's a way to check if the marker exists before sending code to the clients to remove them
> It may not be needed, it might just fail silently if the marker doesn't exist. I can't remember

Reply:
I pushed a change to add the conditional back in. I still think the conditional is unnecessary, however, and will open a separate PR to address that.